packages/client/hmi-client/src/views/DataExplorer.vue

```diff
@@ -1,13 +1,26 @@
 <template>
 	<div class="data-explorer-container">
 		<div class="facets-and-results-container">
-			<facets-panel
-				v-if="viewType === ViewType.LIST"
-				class="facets-panel"
-				:facets="facets"
-				:filtered-facets="filteredFacets"
-				:result-type="resultType"
-			/>
+			<slider content-width="20%" tab-width="48px" direction="left" :is-open="isSliderFacetsOpen">
+				<template v-slot:content>
+					<div class="slider-header content">
+						<IconChevronLeft24 class="slider-header-item" @click="isSliderFacetsOpen = false" />
+						<h4 class="slider-header-item">Facets</h4>
+					</div>
+					<facets-panel
+						v-if="viewType === ViewType.LIST"
+						class="facets-panel"
+						:facets="facets"
+						:filtered-facets="filteredFacets"
+						:result-type="resultType"
+					/>
+				</template>
+				<template v-slot:tab>
+					<div class="slider-header tab">
+						<IconChevronRight24 class="slider-header-item" @click="isSliderFacetsOpen = true" />
+					</div>
+				</template>
```

Comment:
I think this should be handle by the `slider` component as well.
Using the prop direction of opening, we can infer those icons.